### DISCUSSION TITLE
Note that mysqldump has been removed from MariaDB in new docker images, use mariadb-dump instead

### DISCUSSION THREAD

**Discussion by veletron2** [offen/docker-volume-backup#514] [General]
mysqldump has been removed from MariaDB in new docker images, use mariadb-dump instead

https://mariadb.com/kb/en/mysqldump/

This is used where it is desired to export the db to a .sql file prior to backup as per the instructions here:

https://offen.github.io/docker-volume-backup/recipes/#using-mysqldump-to-prepare-the-backup

**Comment by m90** [comment]
Thanks for the heads up. Would you mind opening a PR that updates the docs accordingly?

**Reply by veletron2** [reply]
Apologies, but I have not got a scooby how to do that!!

**Reply by m90** [reply]
Since this is just updating a document, you can probably do this in your browser only by heading over to the source file https://github.com/offen/docker-volume-backup/blob/main/docs/recipes/index.md and hitting the edit button in the top right corner of the document. GitHub will guide you through forking the repository, editing, committing the change and opening the pull request against the original source.

If you want to do this, it'd be much appreciated, but please do not feel obliged or anything, really. I can also add this to my list of things to do at some point, but you know how that works ...

**Comment by m90** [comment]
This is fixed in https://github.com/offen/docker-volume-backup/commit/4799795f0a04a719224b08424872b3453a7458c2
